config/compiler-centos7-gcc73.yaml

```diff
@@ -0,0 +1,19 @@
+compilers:
+- compiler:
+    environment:
+      set:
+        LD_LIBRARY_PATH: /cvmfs/sft.cern.ch/lcg/contrib/gcc/7.3.0binutils/x86_64-centos7/lib64:/cvmfs/sft.cern.ch/lcg/contrib/binutils/2.28/x86_64-centos7/lib:/cvmfs/sft.cern.ch/lcg/releases/binutils/2.28-19981/x86_64-slc6-gcc62-opt/lib:EXTRA_LIBS
+    extra_rpaths:
+    - /cvmfs/sft.cern.ch/lcg/releases/binutils/2.28-19981/x86_64-slc6-gcc62-opt/bin
```

Comment:
Links point to a `slc6` platform while the configuration file is for a compiler built for a `centos7` platform.